README.md

```diff
@@ -3,7 +3,7 @@
 You can use the pubsub_logging.PubsubHandler to publish the logs to [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs/). You can use this module with [the standard Python logging module](https://docs.python.org/2/library/logging.html).
 
 ## Supported version
-Python 2.7 and 3.4 are supported.
+Python 2.7 is supported.
```

Comment:
https://github.com/google/google-api-python-client/issues/3

We're almost there. If httplib2 is uploaded to pypi, it will start working (actually the module works with manually installed httplib2).